packages/Takeover/src/Takeover.js

```diff
@@ -66,24 +61,41 @@ export default function Takeover(props) {
 
   const takeoverOverlay = overlayExtracted || <TakeoverOverlay />;
   const overlayProps = {
-    ...takeoverOverlay.props,
+    zIndex,
     "data-pka-anchor": "takeover.overlay",
-    focusLockOptions,
     hasBackdrop: false,
+    focusLockOptions,
+    ...takeoverOverlay.props,
     isOpen,
     onAfterClose,
     onAfterOpen,
     onClose,
-    zIndex,
   };
 
-  const ariaLabel = a11yText || (headerExtracted ? headerExtracted.props.children : null);
+  function getAriaLabel() {
+    if (a11yText) return a11yText;
+    if (!headerExtracted || typeof headerExtracted.props.children !== "string") {
+      console.error(
+        "Accessibility is important 😇\nThe Takeover requires either an a11yText prop or a Takeover.Header with a string for children."
+      );
+      return null;
+    }
```

Comment:
Perhaps this if block could be better placed outside of this function, maybe a generic displayAccessibilityErrors function (Eventually that could even be its own helper)

I would only expect the actual label text to be returned here to keep it clean...

Reply:
Ideally we could have a custom propType that validates earlier.  But anything we do is really "nice to have" and better than it was.  For now I just need to get this through to finish up a PR.